news/editions/edition3.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: single
+title:  "*Flying*  Edition 3"
+sidebar:
+  nav: news
+redirect_from: /news/latest/
+---
+## Open Source: #BetterTogether
+> This month's issue focuses Spinnaker's strong open source DNA. The OSS community embraces Spinnaker, as it builds a bridge between the collaboration and interoperability story at the core of Linux culture, and the powerful cloud economy that has leveraged it to transform software delivery. Efforts to build native Kubernetes integration highlight this unique strength.
+
+## [The New Age of the K8S Operator](https://www.armory.io/blog/introducing-spinnaker-operator-a-kubernetes-native-blueprint-for-success/)
+Spinnaker Operator, a Kubernetes operator for Spinnaker, has reached Beta. With it installed, you can use kubectl commands to install, deploy, and upgrade any version of Spinnaker through its lifecycle. Read the [doc](https://github.com/armory/spinnaker-operator/blob/release-0.3.x/README.md), try it out, and use the #kubernetes-operator channel in Spinnaker Slack to share your feedback.
+
+## [Join Spinnaker Gardening Days #CommunityHack Virtually on April 9-10](https://github.com/spinnaker-hackathon/gardening)
+As contributors and project newcomers alike gear up for the first Spinnaker Gardening Days, we're collecting and discussing [project ideas](https://github.com/spinnaker-hackathon/gardening/wiki/Project-Ideas). Socializing your ideas at this stage will help shape them into projects, so please visit the Wiki and join or create #gardening-idea Slack channels, and discuss with your peers. Alternatively, visit your favorite SIG's channel or meeting, and ask where you can jump in. Don't forget to [register](https://www.eventbrite.com/e/spinnaker-gardening-days-communityhack-tickets-97845696111) to reserve your ticket!
+
+## [How to Build a Centos RPM Via a Spinnaker "Bake"](https://blog.opsmx.com/how-to-build-bake-centos-rpm-using-spinnaker/)
+While Spinnaker doesn't offer a CentOS baking option by default, this tech blog demonstrates the use of package management and S3 to bake a build as a CentOS image, and deploying an instance of the baked image on AWS cloud. With the RPM built, configuring and executing the bake in Spinnaker requires just a few steps.
```

Comment:
```suggestion
While Spinnaker doesn't offer a CentOS baking option by default, this tech blog demonstrates how to use package management and S3 to bake a build as a CentOS image and deploy an instance of the baked image on AWS cloud. With the RPM built, configuring and executing the bake in Spinnaker requires just a few steps.
```

Reply:
And can that last sentence ("With the RPM built..." be deleted without losing much value? It sounds like it's good information to have in the post, but maybe unnecessary here?